docs/execution-providers/index.md

```diff
@@ -29,9 +29,10 @@ ONNX Runtime supports many different execution providers today. Some of the EPs
 ---|---|---|---
 |Default CPU|[NVIDIA CUDA](../execution-providers/CUDA-ExecutionProvider.md)|[Intel OpenVINO](../execution-providers/OpenVINO-ExecutionProvider.md)|[NUPHAR (tvm/llvm) ](../execution-providers/Nuphar-ExecutionProvider.md) (*preview*)|
 |[Intel DNNL](../execution-providers/oneDNN-ExecutionProvider.md)|[NVIDIA TensorRT](../execution-providers/TensorRT-ExecutionProvider.md)|[ARM Compute Library](../execution-providers/ACL-ExecutionProvider.md) (*preview*)|[Rockchip NPU](../execution-providers/RKNPU-ExecutionProvider.md) (*preview*)|
-||[DirectML](../execution-providers/DirectML-ExecutionProvider.md)|[Android Neural Networks API](../execution-providers/NNAPI-ExecutionProvider.md)|[Xilinx Vitis-AI](../execution-providers/Vitis-AI-ExecutionProvider.md) (*preview*)|
+|[Standalone TVM](../execution-providers/STVM-ExecutionProvider.md) (*preview*)|[DirectML](../execution-providers/DirectML-ExecutionProvider.md)|[Android Neural Networks API](../execution-providers/NNAPI-ExecutionProvider.md)|[Xilinx Vitis-AI](../execution-providers/Vitis-AI-ExecutionProvider.md) (*preview*)|
```

Comment:
Something looks a bit funky with these links

Reply:
So the idea here is that we're positioning Standalone TVM EP as an EP for CPU, GPU and Edge.

Reply:
Ah ok got it, thanks

Reply:
I was misreading the table

Reply:
That'll look like this:
<img width="631" alt="Screen Shot 2021-12-16 at 8 27 17 PM" src="https://user-images.githubusercontent.com/5974488/146435846-1d9b9336-a0cd-406c-9e60-91a66b1f5aac.png">
﻿